plotting/map.py

```diff
@@ -817,7 +817,8 @@ def plot(self):
 
         # Map Info
         self.basemap.drawmapboundary(fill_color=(0.3, 0.3, 0.3), zorder=-1)
-        self.basemap.drawcoastlines(linewidth=0.5)
+        if self.basemap.coastsegs[0]:  # ensure map contains coastline before trying to draw
```

Comment:
Looks like Basemap 1.2.2 can't handle a call to `drawcoastlines` if the map doesn't intersect the ocean anywhere so we need some sort of check like this.